sdk/core/Azure.Core/src/Http/HttpHeader.cs

```diff
@@ -58,6 +58,7 @@ public static class Names
             public static string Accept => "Accept";
             public static string Authorization => "Authorization";
             public static string Range => "Range";
+            public static string XMsRange => "x-ms-range";
```

Comment:
We should move this to a EncryptionConstants for now because it's causing the build to fail since Azure.Core just shipped.